editor/src/components/editor/store/store-hook.ts

```diff
@@ -124,3 +124,52 @@ export type EditorStateContextData = {
 
 export const EditorStateContext = React.createContext<EditorStateContextData | null>(null)
 EditorStateContext.displayName = 'EditorStateContext'
+
+export function useSelectorWithCallback<U>(
+  selector: StateSelector<EditorStore, U>,
+  callback: (newValue: U) => void,
+  equalityFn: (oldSlice: U, newSlice: U) => boolean = utils.shallowEqual,
+  explainMe = false,
```

Comment:
Huh, I guess this infers the type of `explainMe`?

Reply:
yep, it is inferred to boolean. but I should write the type out regardless, to make it consistent with the other parameters

Reply:
fixed in 11f92e59